assets/_sass/custom-viz/boeing/_scrollytelling.scss

```diff
@@ -0,0 +1,29 @@
+/*==============================
+=        Scrollytelling        =
+==============================*/
+
+.label-group {
```

Comment:
For the `.label-group`, `.hidden`, and `.display-none` classes, I would prefix them or otherwise increase the specificity to ensure they don't impact anything outside of the interactive. For example, you could do this:
```scss
#boeignScroll {
  .js-hidden {}
  .js-display-none {}
  .label-group {}
}
```

Note the `js-` prefix, which is used to indicate that those classes are toggled on/off via JavaScript